lib/src/showcase.dart

```diff
@@ -171,10 +171,10 @@ class _ShowcaseState extends State<Showcase> {
 
     if (activeStep == widget.key) {
       _scrollIntoView();
-      if (ShowCaseWidget.of(context)!.autoPlay) {
+      if (ShowCaseWidget.of(context).autoPlay) {
```

Comment:
It would be better if we save `ShowCaseWidget.of(context)` in a variable and use it instead of looking up the tree for its state for so many times!